wdl/transforms/draft3/src/main/scala/wdl/draft3/transforms/wdlom2wom/WorkflowDefinitionElementToWomWorkflowDefinition.scala

```diff
@@ -49,19 +45,19 @@ object WorkflowDefinitionElementToWomWorkflowDefinition {
 
     for {
       linkedGraph <- LinkedGraphMaker.make(nodes = a.graphElements, seedGeneratedValueHandles, typeAliases = a.typeAliases)
-      womGraph <- makeWomGraph(linkedGraph, a.seedNodes, a.workflowName, a.insideAScatter)
+      womGraph <- makeWomGraph(linkedGraph, a.seedNodes, a.workflowName, a.insideAScatter, a.tasks)
     } yield womGraph
   }
 
-  private def makeWomGraph(linkedGraph: LinkedGraph, seedNodes: Set[GraphNode], workflowName: String, insideAScatter: Boolean): ErrorOr[WomGraph] = {
+  private def makeWomGraph(linkedGraph: LinkedGraph, seedNodes: Set[GraphNode], workflowName: String, insideAScatter: Boolean, tasks: ErrorOr[Set[CallableTaskDefinition]]): ErrorOr[WomGraph] = {
```

Comment:
I'd suggest making this `tasks: Set[Callable]` and doing the `ErrorOr` handling in the method that calls this